docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -28,20 +28,23 @@ Changes to the meaning of a text or code example should be done in the main [Eng
 
 The first step for starting a new translation is to check what exists. So far, there are repositories for the following languages:
 
-- [Brazilian Portuguese](https://github.com/gatsbyjs/gatsby-pt-BR)
 - [German](https://github.com/gatsbyjs/gatsby-de)
+- [Spanish](https://github.com/gatsbyjs/gatsby-es)
```

Comment:
Rearranged based on language code to be more neutral (same thing we do for React)